test/integration-tests/shared/protocolHelpers/seaport.ts

```diff
@@ -47,6 +56,19 @@ export type AdvancedOrder = Order & {
   extraData: string // bytes
 }
 
+export enum Side {
+  OFFER,
+  CONSIDERATION,
+}
+
+export type CriteriaResovler = {
+  rderIndex: BigNumber
+  side: Side // enum
+  index: BigNumber
+  identifier: BigNumber
+  criteriaProof: string[] // bytes32[]
+}
```

Comment:
Not used now, but can leave in for the future when criteria orders are supported by OS api